Makefile

```diff
@@ -158,7 +158,7 @@ include network.env
 
 # Build the base command
 # WARNING: WS_ADDR is a sensitive interface and should only be exposed to trusted networks
-BASE_CMD = ./build/bin/go-quai --$(NETWORK) --syncmode full --verbosity 3
+BASE_CMD = ./build/bin/go-quai --$(NETWORK) --syncmode full --verbosity 4
```

Comment:
This needs to be removed. Ideally we close and merge this as well to avoid in the future https://github.com/dominant-strategies/go-quai/pull/679

Reply:
#679 is approved now. Waiting on merge to main, and will rebase on top of that.